tidoop-mr-lib/README.md

```diff
@@ -1,6 +1,240 @@
+* [Jobs reference (in alphabetical order)](#jobs)
+    * [Filter](#filter)
+* [Contact](#contact)
+
+##<a name="whatis"></a>What is tidoop-mr-lib
+
+tidoop-mr-lib is a library of already developed-ready to use [MapReduce](http://research.google.com/archive/mapreduce.html) jobs for [Hadoop](https://hadoop.apache.org), allowing data scientists and engineers in genaral for an easy and quick approach to Big Data analysis.
+
+Available jobs can be used either to transform an input dataset, either to compute a value from it. Usually, these transformations or computations are suited for any general dataset, nevertheless there are NGSI specific MapReduce jobs simplifying a lot the process of analyzing NGSI-like context data. Have a look on this and many other ways of classifying the jobs in the [library details](#details) section.
+
+Finally, it is worth to mention MapReduce jobs available within this library can be combined in a chain of executions, linking the outout of a job to another one's input. Job chaining can be done from the command line, manually running jobs sequentially, using [tidoop-mr-lib-api](../tidoop-mr-lib-api) or using [Apache Oozie](http://oozie.apache.org/). See the [chaining of jobs](#chaining) section for further details.
+
+[Top](#top)
+
+##<a name="mainbuilding"></a>Building
+###<a name="prerequisites"></a>Prerequisites
+
+Maven (and thus Java SDK, since Maven is a Java tool) is needed in order to build tidoop-hadoop-ext.
+
+In order to install Java SDK (not JRE), just type (CentOS machines):
+
+    $ yum install java-1.6.0-openjdk-devel
+
+Remember to export the JAVA_HOME environment variable. In the case of using `yum install` as shown above, it would be:
+
+    $ export JAVA_HOME=/usr/lib/jvm/java-1.6.0-openjdk.x86_64
+
+In order to do it permanently, edit `/root/.bash_profile` (`root` user) or `/etc/profile` (other users).
+
+Maven is installed by downloading it from [maven.apache.org](http://maven.apache.org/download.cgi). Install it in a folder of your choice (represented by `APACHE_MAVEN_HOME`):
+
+    $ wget http://www.eu.apache.org/dist/maven/maven-3/3.2.5/binaries/apache-maven-3.2.5-bin.tar.gz
+    $ tar xzvf apache-maven-3.2.5-bin.tar.gz
+    $ mv apache-maven-3.2.5 APACHE_MAVEN_HOME
+
+[Top](#top)
+
+###<a name="building"></a>Building tidoop-mr-lib and its dependencies
+
+Building the project can be done by including the dependencies in the package (**recommended**):
+
+    $ git clone https://github.com/telefonicaid/fiware-tidoop.git
+    $ cd fiware-tidoop/tidoop-mr-lib
+    $ git checkout <branch>
+    $ APACHE_MAVEN_HOME/bin/mvn clean compile assembly:single
+
+or not:
+
+    $ git clone https://github.com/telefonicaid/fiware-tidoop.git
+    $ cd fiware-tidoop/tidoop-mr-lib
+    $ git checkout <branch>
+    $ APACHE_MAVEN_HOME/bin/mvn package
+
+where `<branch>` is `develop` if you are trying to install the latest features or `release/x.y.z` if you are trying to install a stable release.
```

Comment:
Fixed in https://github.com/telefonicaid/fiware-tidoop/commit/bb5e6d0ae4b5ea375c06376f7c17378cb6f1dd3f